sound/soc/sof/intel/hda-stream.c

```diff
@@ -465,8 +465,8 @@ irqreturn_t hda_dsp_stream_threaded_handler(int irq, void *context)
 {
```

Comment:
ASoC: SOF: Intel: use snd_sof_pcm_period_elapsed

Switch to a wrapper function which schedules the actual call of snd_pcm_period_elapsed after the current IPC is completed